components/Tabs/src/TabBarView/MDCTabBarView.m

```diff
@@ -15,9 +15,15 @@
 #import "MDCTabBarView.h"
 #import "private/MDCTabBarViewItemView.h"
 
+// KVO contexts
+static char *const kKVOContextMDCTabBarView = "kKVOContextMDCTabBarView";
+
 /** Minimum (typical) height of a Material Tab bar. */
 static const CGFloat kMinHeight = 48;
 
+static NSString *const kImageKeyPath = @"image";
+static NSString *const kTitleKeyPath = @"title";
```

Comment:
I wonder if it makes sense to use more specific names? In case we want to add other related properties?  Could there be more that one kind of image (like leading/trailing, etc).

Reply:
These are key paths, so they are based on the property names relative to the objects being observed (`UITabBarItem` objects).